server/app.js

```diff
@@ -197,6 +198,7 @@ app.use(passport.session())
 
 app.use(`${basePath}/`, indexRouter)
 app.use(`${basePath}/`, chartsRouter)
+app.use(`${basePath}/`, userRouter)
```

Comment:
should we put this in the routes constants, something like `routes.root`?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/241/commits/546e1165ef9fb0c12b8a04131021f34d2cbcb613